Update docs for new Kendo JS and CSS changes (#1680)
telerik/reporting-docs: embedding-reports/display-reports-in-applications/web-application/html5-report-viewer/manual-setup.md

# Manual Setup of the HTML5 Report Viewer in Web Applications in the .NET Framework

> In this topic, we demonstrate how to manually add the HTML5 Report Viewer to an HTML page and to display a report. The approach that we use here allows for full control over the configuration. 

Reviewer: ```suggestion
In this article, you will learn how to manually add the HTML5 Report Viewer to an HTML page and display a report. This approach allows for full control over the configuration. 
```

1. Add a reference to jQuery in the `<head>` element:

	````HTML
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
````

	>jQuery must be loaded before creating the viewer object.jQuery must be loaded only once on the page.

> 1. Add references to the Kendo UI SASS styles in the `<head>` element:

Reviewer: ```suggestion
1. Add references to the Kendo UI Sass styles in the `<head>` element:
```

	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.panelbar.min.js"></script>
	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.tooltip.min.js"></script>
	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.treeview.min.js"></script>
	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.touch.min.js"></script>
	kendo.mobile.min.js - optional, if gestures/touch support is required
	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.mobile.min.js"></script>
	kendo.all.min.js or kendo.web.min.js can be used as well if Kendo is used outside the report viewer
	<script src="https://kendo.cdn.telerik.com/{{kendothemeversion}}/js/kendo.web.min.js"></script>
	-->
````

1. Add a `<div>` element to the `<body>` element that will serve as a placeholder for the viewer’s widget. The `<div>` element's ID attribute serves as a key(Id) for the viewer object. Its content (*loading...*) will be displayed while the viewer’s content is being loaded (from the template):

	````HTML
<div id="reportViewer1">
		loading...
	</div>
````

> 1. Add the following script element at the bottom of the `<body>` element and create the HTML5 Report Viewer widget for the reportViewer1 `<div>` element that we added above:

Reviewer: ```suggestion
1. Add the following script element at the bottom of the `<body>` element and initialize the HTML5 Report Viewer widget through the reportViewer1 `<div>` element that we added above:
```
